Sources_v3/Query/Sorting.swift

```diff
@@ -102,3 +102,53 @@ extension ChannelListSortingKey {
         return nil
     }
 }
+
+public enum UserListSortingKey: SortingKey {
```

Comment:
Since this is public API, can you add documentation please? I also noticed we don't documentation for `ChannelListSortingKey`, can you add documentation there, too? It will be almost identical.